### PR TITLE
ENYO-6352: Fix Notification to support 3 full-width buttons in one line

### PR DESCRIPTION
**Pull request by Djspaceg**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
3 max-width buttons don't fit in a wide `Notification`.

### Resolution
The width of `Notification`is now based on the size of 3 max-width buttons. (It's wider now.)

### CODE REVIEW

File: packages/moonstone/styles/variables.less [hunk 1]
```diff
@@ -470,8 +470,8 @@
 @moon-notification-padding-left-right: 39px;
 @moon-notification-padding-top-bottom: 24px;
 @moon-notification-min-width: 300px;
-@moon-notification-max-width: ((@moon-notification-border-width * 2) + (@moon-notification-padding-left-right * 2) + (2 * ((@moon-button-small-h-margin * 2) + @moon-button-small-max-width)));  // Attempt to calculate the width of all of the buttons and the pieces of Notification that influence the width to get the maximum size
-@moon-notification-wide-width: 1287px;
+@moon-notification-max-width: ((@moon-notification-border-width * 2) + (@moon-notification-padding-left-right * 2) + (2 * ((@moon-button-small-h-margin * 2) + @moon-button-small-max-width)) + 12px);  // Attempt to calculate the width of all of the buttons and the pieces of Notification that influence the width to get the maximum size
+@moon-notification-wide-width: ((@moon-notification-border-width * 2) + (@moon-notification-padding-left-right * 2) + (3 * ((@moon-button-small-h-margin * 2) + @moon-button-small-max-width)) + 12px);
```
Comment (webOS101) [inline packages/moonstone/styles/variables.less]: Could we make this something like:
```
@moon-notification-wide-width: @moon-notification-max-width + ((@moon-button-small-h-margin * 2) + @moon-button-small-max-width))
```
(Or, perhaps even collapse `(@moon-button-small-h-margin * 2) + @moon-button-small-max-width)` into a `@moon-button-small-full-width`)

Comment (Djspaceg) [reply packages/moonstone/styles/variables.less]: I'd like to discorage the creation (and maintinence) of specialized variables that aren't directly used by the components' LESS files.

Comment (webOS101) [reply packages/moonstone/styles/variables.less]: That makes sense.  What about the meatier part of the comment?

Comment (Djspaceg) [reply packages/moonstone/styles/variables.less]: I'm less excited about that approach as it assumes that the metrics for normal vs wide are the same. With this approach it's currently very clear that they're identical with the only difference being the amount of buttons. That approach also abstracts what's going on which means you need to understand what's going on in the normal version before cloning part of that into the wide. Imagine in the future we need to change the amount of buttons. Now that line must change drasticly by adding in an additional calculation. The current approach allows you to just change 1 number. Similar to what would happen if wide mode suddenly had smaller paddings. The current approach you'd just update the padding and be done. With the composite approach you'd have to reverse engineer where those numbers are different and figure out how to get it all right again.

**tl;dr**: It's better for maintainability if we don't do that.

Comment (webOS101) [reply packages/moonstone/styles/variables.less]: Works for me!
